### PR TITLE
feat: add EventOwnerChanged and EventRootChanged

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

Add `EventOwnerChanged` and `EventRootChanged`.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: proto/lbm/collection/v1/event.proto [hunk 1]
```diff
@@ -299,3 +299,23 @@ message EventDetached {
   // token being detached.
   string subject = 4;
 }
+
+// EventOwnerChanged is emitted when the owner of token is changed by operation applied to its ancestor.
+//
+// Since: 0.46.0 (finschia)
+message EventOwnerChanged {
```
Comment (zemyblue) [inline proto/lbm/collection/v1/event.proto:306]: Is there no changed owner information in this event?

Comment (0Tech) [reply proto/lbm/collection/v1/event.proto:306]: Yes, owner was not included in the event on Daphne.

Comment (zemyblue) [reply proto/lbm/collection/v1/event.proto:306]: If so, can I understand that it was not added for backwards compatibility?

**Review by zemyblue — APPROVED** [review]

File: proto/lbm/collection/v1/event.proto (hunk 1 above)
Comment (0Tech) [reply proto/lbm/collection/v1/event.proto:306]: Adding a new field would not break the compatibility, but there is no reason to add it.

**Review by dudong2 — APPROVED** [review]
LGTM

**Pull request merged**